taxonomies/categories.txt

```diff
@@ -95616,7 +95661,7 @@ fr:Civet de tangue
 fr:Civet de zourites
 wikidata:en:Q16540075
 
-
+<en:Meals
```

Comment:
Most products in this category are not meals: https://world.openfoodfacts.org/category/Cr%C3%AApes%20and%20galettes

Reply:
where would you put it then ?